### ISSUE TITLE
[Package]: 7-Zip ZS

### ISSUE THREAD

**Issue by Cyberavater** [termux/termux-packages#18000]
### Why is it worth to add this package?

The official 7zip package, which is already added in termux repo doesn't have zstd support. This one however has zstd support and some additional codecs
https://github.com/mcmilk/7-Zip-zstd#version-information

### Home page URL

https://github.com/mcmilk/7-Zip-zstd

### Source code URL

https://github.com/mcmilk/7-Zip-zstd

### Packaging policy acknowledgement

- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied in case of violation.

**Comment by TomJo2000** [comment]
This package probably does not meet the guidelines outlined in the packaging policy for the following (main) reasons.
- Not serving duplicated functionality.
As you have stated yourself, this package is a fork of the already packaged 7zip with some added support for additional compression schemes (largely already available through other packages)

- Software available in major Linux distributions.
7zip-zstd is packaged in exactly [3 major repositories](https://repology.org/project/7zip-zstd/versions).
  - Scoop
  - Chocolatey and
  - winget

All three are packaging systems for Windows, where access to multiple compression utilities is less of a given.
In conclusion, while this project does add desirable additional functionality to 7zip it is not widely distributed, nor does provide unique functionality.

It is unlikely to be a good fit for inclusion in the Termux repositories.
Although maybe [the TUR](https://github.com/termux-user-repository/tur/) would be interested in packaging it.

**Comment by Cyberavater** [comment]
> Software available in major Linux distributions.
>7zip-zstd is packaged in exactly [3 major repositories](https://repology.org/project/7zip-zstd/versions).

>Scoop
>Chocolatey and
>winget

Not to be rude but those are not *Linux distributions*


>largely already available through other packages
Maybe you are correct. we may use https://packages.termux.dev/apt/termux-main/pool/main/p/p7zip/ (I have not tested it yet, but it should since it seems to be building from https://github.com/p7zip-project/p7zip as https://github.com/termux/termux-packages/blob/ea19ac641cbe6f51666ea4909016973f81278bd8/packages/p7zip/build.sh#L7C44-L7C44) However, 7-Zip ZS is more actively worked on and updated (with latest codecs while p7zip codecs are old)  so that should be a plus.

You can close it if you don't see it fit, tho.

**Comment by TomJo2000** [comment]
`p7zip` is a bit of a special case actually.
Mainline `7zip` has only really started supporting Linux as of Version `21.01`.
So most Linux distributions instead packaged the `p7zip` fork to provide support for `.7z` files.

The same is true for Termux.
The `7zip` package largely supersedes the `p7zip` package, but there isn't really a reason to remove it either.

---

> You can close it if you don't see it fit, tho.

I am not a maintainer, I do not have the power to close issues other than my own.
I'm just a regular contributor.
